### PR TITLE
Add requirement for Paper and Equator app

### PR DESCRIPTION
**Pull request by MariuszWisniewski**
Adding libraries for paper&equator application.

### CODE REVIEW

**Comment by MariuszWisniewski** [comment]
@Zhebr ?

File: package.json.j2 [hunk 1]
```diff
@@ -21,6 +21,22 @@
...
+    "geolib": "~2.0.14", 
+    "gifencoder": "1.0.6", 
+    "googlemaps": "0.1.20", 
+    "lwip": "0.0.8", 
+    "mkdirp": "0.5.0", 
+    "png-file-stream": "1.0.0", 
+    "polyline": "0.0.3", 
+    "q": "1.1.2", 
+    "winston": "^2.2.0", 
+    "winston-daily-rotate-file": "^1.0.1"
```
Comment (kotchaosu) [inline package.json.j2]: missing comma

**Comment by kotchaosu** [comment]
```
npm WARN optional Skipping failed optional dependency /machinepack-forecast.io/babel/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@0.3.8
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.12
npm WARN superagent-retry@0.4.0 requires a peer of superagent@~0.19.0 but none was installed.
npm WARN nodejs-codebox@0.0.1 No repository field.
npm WARN nodejs-codebox@0.0.1 No license field.
npm ERR! Linux 4.4.6-201.fc22.x86_64
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v5.6.0
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE

npm ERR! lwip@0.0.8 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lwip@0.0.8 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the lwip package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs lwip
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls lwip
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/syncano/v1.0/npm-debug.log
```

**Comment by kotchaosu** [comment]
I wondered why your config without comma was correct - it wasn't, but failed silently:

```
npm info it worked if it ends with ok
npm info using npm@3.6.0
npm info using node@v5.6.0
npm info attempt registry request try #1 at 2:29:08 PM
npm http request GET https://registry.npmjs.org/fsevents
npm http 304 https://registry.npmjs.org/fsevents
npm info lifecycle undefined~preinstall: undefined
npm info linkStuff !invalid#1
npm info lifecycle undefined~install: undefined
npm info lifecycle undefined~postinstall: undefined
npm info lifecycle undefined~prepublish: undefined
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.12
npm WARN optional Skipping failed optional dependency /machinepack-forecast.io/babel/chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@0.3.8
npm WARN Failed to parse json
npm WARN Unexpected token 'd' at 41:6
npm WARN     "dk-machinepack-facebookads": "*",
npm WARN      ^
npm WARN File: /home/syncano/v1.0/package.json
npm WARN superagent-retry@0.4.0 requires a peer of superagent@~0.19.0 but none was installed.
npm WARN v1.0 No description
npm WARN v1.0 No repository field.
npm WARN v1.0 No README data
npm WARN v1.0 No license field.
npm info ok
```

**Comment by MariuszWisniewski** [comment]
Might replace `lwip` with `jimp`, closing for now.

**Comment by kotchaosu** [comment]
neat - let's merge!

**Pull request merged**